udfs/migration/snowflake/test_cases.yaml

```diff
@@ -0,0 +1,14 @@
+  
+factorial:
+  - test:
+    input: 0
+    expected_output: 1.0
+  - test:
+    input: 1
+    expected_output: 1.0
+  - test:
+    input: 5
+    expected_output: 120
+  - test:
+    input: 10
+    expected_output: 3628800
```

Comment:
Should these expected outputs also have a `.0` since the return type is NUMERIC?

Reply:
BQ does type conversion and the test passes. I can add `.0` to emphasize that the return type is NUMERIC